include/cpr/accept_encoding.h

```diff
@@ -0,0 +1,41 @@
+#ifndef CPR_ACCEPT_ENCODING_H
+#define CPR_ACCEPT_ENCODING_H
+
+#include <curl/curlver.h>
+#include <initializer_list>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace cpr {
+
+enum class AcceptEncodingMethods {
+  identity,
+  deflate, 
+  zlib, 
+  gzip,
+};
+
+class AcceptEncoding {
+  public:
+    std::map<AcceptEncodingMethods, std::string> MethodsStringMap { 
```

Comment:
Can't this map be `static` and `const`?

Reply:
I would investigate it and find a best practice for that